frontend/src/components/_common/Field/index.tsx

```diff
@@ -0,0 +1,22 @@
+import type { InputHTMLAttributes } from 'react';
+
+import Input from '../Input';
+import { s_description, s_field, s_label } from './Field.styles';
+
+interface FieldProps {
+  label: string;
+  description?: string;
+  inputProps?: InputHTMLAttributes<HTMLInputElement>;
+}
+
+export default function Field({ label, description = '', inputProps }: FieldProps) {
```

Comment:
[P3]

```suggestion
interface FieldProps extends InputHTMLAttributes<HTMLInputElement> {
  label: string;
  description?: string;
}

export default function Field({ label, description = '', ...props }: FieldProps) {
```

이렇게 표현해볼 수도 있을 것 같습니다! 가벼운 제안입니다!

Reply:
오옷!!! 낙타🐫 좋은 의견이네요! `InputHTMLAttributes`를 확장하는 것이 훨씬 깔끔한 것 같습니다. 반영하였습니다~